### ISSUE TITLE
Qwiic RFID acks on every I2C address

### ISSUE THREAD

**Issue by nseidle** [sparkfun/SparkFun_Qwiic_RFID_Arduino_Library#4]
Why is the ATtiny ack'ing on every I2C address? This is causing issues talking to other I2C devices on my bus.

![image](https://user-images.githubusercontent.com/117102/71218007-2679e300-227d-11ea-8931-c8f402eeab7c.png)

Here's the I2C scanner sketch.

#include <Wire.h>

void setup()
{
  Serial.begin(115200);
  Wire.begin();

  while (1)
  {
    byte error, address;
    int nDevices;

    Serial.println("Scanning...");

    nDevices = 0;
    for (address = 1; address < 127; address++ )
    {
      // The i2c_scanner uses the return value of
      // the Write.endTransmisstion to see if
      // a device did acknowledge to the address.
      Wire.beginTransmission(address);
      error = Wire.endTransmission();

      if (error == 0)
      {
        Serial.print("I2C device found at address 0x");
        if (address < 16)
          Serial.print("0");
        Serial.print(address, HEX);
        Serial.println("  !");

        nDevices++;
      }
      else if (error == 4)
      {
        Serial.print("Unknown error at address 0x");
        if (address < 16)
          Serial.print("0");
        Serial.println(address, HEX);
      }
    }
    if (nDevices == 0)
      Serial.println("No I2C devices found\n");
    else
      Serial.println("done\n");

    delay(5000);           // wait 5 seconds for next scan
  }

}

void loop()
{
 
}

**Comment by nseidle** [comment]
I'm using Arduino 1.8.10 and Artemis with the latest Qwiic RFID lib.

**Comment by edspark** [comment]
It's a very enthusiastic chip just trying to to please? 

Hur hur, I'll give a look see sometime tomorrow, if not, early next week.

**Comment by edspark** [comment]
A preliminary once over of the issue and I've found that with an Redboard, I don't see the issue:
![Scan with Uno](https://user-images.githubusercontent.com/18684691/71834915-96ee7580-306d-11ea-9d23-993ae1315fe1.jpg)

I do see the issue with the Artemis but in closer inspection with a logic analyzer I'm finding that the ACKs in the serial terminal are not lining up with the behavior of the transmissions: 

Here's an ACK at adress 0x7D:
![clear ACK](https://user-images.githubusercontent.com/18684691/71835059-eaf95a00-306d-11ea-8a1b-b80ea6a97cb2.jpg)

Here's a NAK at address 0x7C - repeated for every address that is **not** 0x7D:
![Clear NAK](https://user-images.githubusercontent.com/18684691/71835064-ef257780-306d-11ea-8295-8e77f2de180e.jpg)

List of addresses being NAK'ed and the single address been ACK'ed:
![logic_analyzer_ACK](https://user-images.githubusercontent.com/18684691/71835070-f0ef3b00-306d-11ea-82a7-5ac3de2800f6.jpg)

I'm not seeing any error in the logic of the Arduino Sketch, so I think the next step may need to be to hook up the I2C lines to a scope.

**Comment by nseidle** [comment]
I don't think we have this issue (Artemis doing weird things) on any other ATtiny based qwiic device so I'm nervous.

As an aside, I've been dealing with ublox modules doing evil things with I2C setup times on the Artemis. I think it's related to bus capacitance. We recently added a setPullups() feature to Artemis so for giggles, try adding **Wire.setPullups(0); //Disable internal I2C pullups** to your test sketch and see how the logic traces change (if any).
